contents/node/nodejs.md

```diff
@@ -0,0 +1,208 @@
+# Introduction to Node.js
+
+Author: Rachael Sim
+
+# Overview
+
+_This chapter assumes that the reader is familiar with JavaScript and asynchronous programming. If you are not familiar with asynchronous programming, a good resource to checkout is the [asynchronous programming section of the You Don't Know JS guide ](https://github.com/getify/You-Dont-Know-JS/blob/master/async%20&%20performance/README.md#you-dont-know-js-async--performance) as asynchronous programming is key in Node._
+
+*Node.js* is a JavaScript runtime built on Chrome’s V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.
```

Comment:
Mention where it fits into the Web technology landscape? If not, the reader might assume it is a front-end technology. Giving a few example scenarios of how node is used (for an architecture point of view) can be useful too. e.g., a web app written using node.js, a non-web app using node etc. Can be real or hypothetical